scripts/deployDocs.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# package docs and deploy
+
+DOC_PATH='./docs'
+VERSION=`cat $DOC_PATH/source/conf.py | egrep 'release\s?=' | awk '{print $3}' | sed -e "s/^'//" -e "s/'$//"`
+PROJECT_NAME=`cat $DOC_PATH/source/conf.py | egrep 'project\s?=' | awk '{print $3}' | sed -e "s/^'//" -e "s/'$//"`
+SOURCE_FILES="$DOC_PATH/build/html/"
+PACKAGE_NAME="docs_${PROJECT_NAME}_${VERSION}"
+DEPLOY_FILENAME="${DOC_PATH}/${PACKAGE_NAME}.tar.gz"
+
+DEPLOY_USER=docs_deploy
+
+if [ ! -z "$1" ]; then
+    DEPLOY_SERVER="$1"
+fi
+
+if [ ! -z "$2" ]; then
+    DEPLOY_USER="$2"
+fi
+
+echo "building docs package $PACKAGE_NAME"
+
+# install dev packages for doc build
+pip install -e .[dev] -U tox-travis
+
+# make the docs from source
+make docs
+
+# package into a tar.gz file for deployment
+tar -czvf "$DEPLOY_FILENAME" "$SOURCE_FILES"
+
+if [ ! -z "$DEPLOY_SERVER" ]; then
+    DEPLOY_BUILD_URL="http://${DEPLOY_SERVER}/docs_build"
```

Comment:
As there is a Let's Encrypt cert for this server is there any reason to use http instead of https?

Reply:
No idea, but yes we can use https  & Let's Encrypt